### DISCUSSION TITLE
Seasons Discussion Questions or Suggestions Submissions

### DISCUSSION THREAD

**Discussion by Zageron** [flashflashrevolution/rrr#63] [Ideas]
Submit your questions, suggestions, or comments here before the meeting and we'll address them.

**Comment by maxproske** [comment]
I'm committed to making a shitty time-jailed leaderboard and top 100, with no pressure on the R^3 team to implement engine support for the first season.

**Reply by maxproske** [reply]
That being said, it would be a welcome surprise ;)

**Comment by wcheung-code** [comment]
**Suggestion:**

I think it would be worthwhile to add a player's skill rating line graph to improve the user's experience of the player in a given season. Specifically, it would be great to see a graph similar to one shown in [this link under tier graph next to the blue update button](https://na.op.gg/summoner/userName=Doublelift) (screenshot provided below for reference).

![Capture](https://user-images.githubusercontent.com/22122355/118217604-cf614900-b443-11eb-82bd-40fc888107cc.PNG)

At every three months, the first data point would carry a 0.0 skill rating for the first day of the season. When this player plays any new song and acquire skill rating from receiving a high score, this data point will continuously be updated and is set to record at the end of the day. This proceeds until the end of the season where the skill ratings reset in preparation for next season.
